src/frontend/query_protocol.ml

```diff
@@ -133,6 +133,12 @@ type _ t =
        | `Not_found of string * string option
        | `No_documentation
        ] t
+  | Syntax_document
+    : Msource.position
+    ->[ `Found of string
+      | `Invalid_context
+      | `No_documentation
```

Comment:
What difference do you make between these two return values ?
It looks like we never return `` `Invalid_context``.

Reply:
I think I need to remove `Invalid_context`. It could have made sense if I was checking that the position actually contains an interesting node but since I am not currently doing that, I think I'll best remove it.

Great catch